JenkinsfileCloudfrontInvalidation

```diff
@@ -0,0 +1,26 @@
+node('linux && docker') {
+  def commitHash = params.packageName.substring(params.packageName.lastIndexOf('/') + 1)
```

Comment:
I don't think that's really needed.

We can just always checkout master, since what we are interested in is the latest version of the `invalidate-cloudfront.js` script available on master, not the code associated with the package created.